src/compiler/transformers/module/module.ts

```diff
@@ -820,6 +831,14 @@ namespace ts {
             let statements: Statement[];
             let variables: VariableDeclaration[];
             let expressions: Expression[];
+
+            const parseTreeNode = getParseTreeNode(node);

```

Comment:
Why wouldn't we emit __esModule here? Perhaps I'm not understanding when we do/don't want to emit this marker.